include/mbedtls/error.h

```diff
@@ -114,6 +114,48 @@ extern "C" {
 #define MBEDTLS_ERR_ERROR_GENERIC_ERROR       -0x0001  /**< Generic error */
 #define MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED -0x006E  /**< This is a bug in the library */
 
+/**
+ * \brief Combines a high-level and low-level error code together.
```

Comment:
Please document that it's ok for `high`, `low` or both to be 0.

Reply:
Done.